[GER] Add definitions for numbers from 10 to 12

etwasanderes2 opened the pull request:
These numbers are typically written out in german.

etwasanderes2 committed:
[GER] Add definitions for numbers from 10 to 12

These numbers are typically written out in german.

H1ghBre4k3r commented on german.hpp:
```diff
@@ -38,6 +38,9 @@
 #define neun 9
+#define zehn 10
+#define elf 11
+#define zwölf 12
```
Maybe rename to `zwoelf`.

H1ghBre4k3r replied:
Or does this work in C++? I have no idea.

etwasanderes2 committed:
[GER] Basic arithemtic operations

We can still consider modifying the current words to make the resulting
experssions sound more like german text. For example:
```cpp
ganzZahl x sei bitweiseNegiert neun linksverschobenUm zwei;
```
instread of
```cpp
ganzZahl x sein bitweiseNegation 9 linksverschiebung zwei;
```

Also, the ?: operator is still missing.

etwasanderes2 committed:
[GER] Added ?: operator

Notice that `sonst` gets translated to `:`, while `ansonsten` gets
translated to `else`

etwasanderes2 committed:
[GER] temporary half-done stuff

A test file with errors, also we want consistent camel/snake case.
But I need to stop now.

etwasanderes2 committed:
[GER] forgot to add test file
